internal/task_documentation/task_documentation.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
```

Comment:
Wouldn't it be simpler if we had a symlink pointing to the `tasks` directory from `docs`. This is what we do in [ec-policies](https://github.com/hacbs-contract/ec-policies/tree/main/antora/docs) for `policies`?

Reply:
I suppose that prompts the question: are we dogmatic that all tasks will reside only under the `task` directory? If so, I think that the logic in this file can be simplified.

Reply:
Yeah, I think that's needed for the Artifact Hub (cc @joejstuart)

Reply:
I think we can decide this. Tasks in the tasks directory makes sense to me.